src/purchases.ts

```diff
@@ -716,4 +731,17 @@ export default class Purchases {
         RNPurchases.setCreative(creative);
     }
 
+    /**
+     * Check if billing is supported for the current Play user (meaning IN-APP purchases are supported)
+     * and optionally, whether a list of specified feature types are supported. This method is asynchronous
+     * since it requires a connected BillingClient.
+     * @param feature An array of feature types to check for support. Feature types must be one of 
+     *       [BILLING_FEATURE]. By default, is an empty list and no specific feature support will be checked.
+     * @returns {Promise<Boolean>} promise with boolean response
```

Comment:
this copy is very Android-specific, but the implementation works for both iOS and Android. 
we should specify that: 
- this works for the current `user` (not just Play user)
- Google Play only on Android (i.e.: no Amazon)
- works on iOS as well
- features array is only needed for Android purposes, you can pass anything for iOS